nspanel_esphome.yaml

```diff
@@ -361,7 +361,7 @@ globals:
   - id: temperature_correction_global
     type: float
     restore_value: true
-    initial_value: '0.0'
+    initial_value: '-1.2'
```

Comment:
Could you please explain why having `-1.2` and not `0.0` as initial correction for temperature?

Reply:
Really sorry please reject, 1st time using github, this was only a change I was playing around with for my device